### PR TITLE
feat/FOROME-1552: dropdown component

### PR DESCRIPTION
**Pull request by QSdmitrioul**
[Task](https://quantori.atlassian.net/browse/FOROME-1552)

### CODE REVIEW

File: src/components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  extraOptions = [],
+  showCheckboxes = false,
+  clearAll,
+  placeholder = 'Select value',
+  variant = 'primary',
+  offset = popoverOffset,
+  renderItem,
+  renderChosen,
+  prepend = <Icon name="Folder" />,
+  append = <Icon name="Loupe" />,
```
Comment (AndreyUstyumenko) [inline src/components/dropdown/dropdown.tsx]: Why those two have default values? They should be completely optional, and the basic dropdown should have only text/placeholder on the left and an arrow on the right.

**Comment by AndreyUstyumenko** [comment]
If I got it right, it works only in multiselect mode. Multiselect is a good option but number one priority is to implement the very basic behavior. The very default mode should work as html <select> does. For example try to replace this component with the new dropdown. But as an option, we could still be able to pass a prop 'isMulti' for using the multiselect mode.
Screen Shot 2022-08-31 at 03 29 04

**Comment by AndreyUstyumenko** [comment]
Screen Shot 2022-08-31 at 03 29 04

File: src/components/dropdown/components/dropdown-menu/dropdown-menu.module.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.dropdown-menu {
+  border-radius: 4px;
+  padding: 8px 0;
+  box-shadow: theme("boxShadow.card");
+  display: flex;
+  flex-direction: column;
+  width: fit-content;
+  min-width: 260px;
```
Comment (Lapk1n) [inline src/components/dropdown/components/dropdown-menu/dropdown-menu.module.css]: Change the order of these properties from most important to less important

File: src/components/dropdown/dropdown.tsx (hunk 1 above)
Comment (QSdmitrioul) [reply src/components/dropdown/dropdown.tsx]: Fixed

File: src/components/dropdown/components/dropdown-menu/dropdown-menu.module.css (hunk 2 above)
Comment (QSdmitrioul) [reply src/components/dropdown/components/dropdown-menu/dropdown-menu.module.css]: Fixed

**Comment by QSdmitrioul** [comment]
Fixed all comments and implement it in FR

**Review by Lapk1n — COMMENTED** [review]
Screenshot 2022-09-01 at 11 07 49
1. In the XL-ds this select is disabled, but it's not obvious until you click on it

**Comment by Lapk1n** [comment]
Screenshot 2022-09-01 at 11 09 10
1. Changing inheritance mode invokes the select width changing, I think its not good.

**Comment by QSdmitrioul** [comment]
fixed

**Review by Lapk1n — APPROVED** [review]
Screenshot 2022-09-01 at 20 51 16
Can you align this input with the dropdown in accordance with the market? Besides, dropdowns in this panel have a 28px height instead of 32

**Comment by QSdmitrioul** [comment]
We have one size dropdown in ui kit. It seems that it is something with this panel design. As for aligning, I can't do that, cause in maket shown only value with one digit, but we sometimes have two and in this case it doesn't have enough space.

**Comment by Lapk1n** [comment]
If the dropdown height is a constant we can change the height of the `<InputNumeric/>` to h-8, add him max-w-[75px] just like in the dropdown below and decrease the pl of its parent div. The max value of this input is 99 and there is enough space to show it

**Comment by AndreyUstyumenko** [comment]
color should be white 
Screen Shot 2022-09-05 at 13 17 01

**Comment by QSdmitrioul** [comment]
Fixed

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**